lib/server/routes/export.js

```diff
@@ -323,7 +323,7 @@ const exportHandler = (request, response) => {
       // Decide whether to download or not chart file
       if (!body.noDownload) {
         response.attachment(
-          `${request.params.filename || 'chart'}.${type || 'png'}`
```

Comment:
have we officially supported `filename` as a query param previously? If so, we should do `request.params.filename || request.body.filename || 'chart'`.

Reply:
You're right, I did it this way because I did not find support for the `request.params.filename` anywhere in our documentation. But I agree 100% that just for safety it might be better to do it this way.